docs/BuildDigits.md

```diff
@@ -49,39 +49,10 @@ sudo pip install -e $DIGITS_HOME
 
 # Starting the server
 
-You can run DIGITS in two modes:
-
-### Development mode
-
 ```sh
-./digits-devserver
-```
-
-Starts a development server (werkzeug backend) at `http://localhost:5000/`.
-```
-$ ./digits-devserver --help
-usage: __main__.py [-h] [-p PORT] [-d] [--version]
-
-DIGITS development server
-
-optional arguments:
-  -h, --help            show this help message and exit
-  -p PORT, --port PORT  Port to run app on (default 5000)
-  -d, --debug           Run the application in debug mode (reloads when the
-                        source changes and gives more detailed error messages)
-  --version             Print the version number and exit
+python -m digits
```

Comment:
Actually, I think I'll leave `digits-devserver` in there for now and keep telling people to use it. Next step can be:

* PyPI support
* Console entry point (it does work now)